src/DynamicRupture/Output/Base.hpp

```diff
@@ -14,10 +14,11 @@ class Base {
   public:
   virtual ~Base() = default;
 
-  void setInputParam(const YAML::Node& inputData, const MeshReader& mesher) {
+  void setInputParam(const YAML::Node& inputData, MeshReader& userMesher) {
     using namespace initializers;
 
     ParametersInitializer reader(inputData);
+    mesher = &userMesher;
```

Comment:
`mesher` sound strange. For me gmsh or simmetrix are meshers. Why not meshReader?

Reply:
done